modules/RTC/RTCUtils.js

```diff
@@ -814,10 +814,7 @@ class RTCUtils extends Listenable {
 
             this.getStreamID = ({ id }) => id;
             this.getTrackID = ({ id }) => id;
-        } else if (browser.isChrome() // this is chrome < 61
-                || browser.isOpera()
-                || browser.isNWJS()
-                || browser.isElectron()
+        } else if (browser.isChromeLike() // this is chrome < 61
```

Comment:
is this comment still accurate?

Reply:
Yes: we use that code for anything which uses the Chrome engine and its version is < 61.